wolfcrypt/test/test.c

```diff
@@ -27625,6 +27625,22 @@ WOLFSSL_TEST_SUBROUTINE wc_test_ret_t hpke_test(void)
     #endif
 #endif
 
+/* added this code to hit code coverage on lines 219-225 on hpke.c*/
+#if defined(HAVE_CURVE448) && \
+    (defined(WOLFSSL_SHA384) || defined(WOLFSSL_SHA512))
+    /* have_curve448 */
+    ret = wc_HpkeInit(hpke, DHKEM_X448_HKDF_SHA512, HKDF_SHA512,
+        HPKE_AES_128_GCM, NULL);
+    printf("ERROR RIGHT HERE %d\n",ret);
```

Comment:
This call will fail with BAD_FUNC_ARG because Curve448 is not implemented in hpke.c.
Please don't include wolfssl-5.7.0 folder. Only test.c changed.

Reply:
I removed the wolfssl-5.7.0 folder.

Reply:
Ok I believe I implemented those fixes correctly.